packages/map/test/enzyme.test.js

```diff
@@ -29,40 +29,4 @@ describe('To test the Map Box Component.', () => {
   it('test <Map /> Component render', () => {
     expect(wrapper.find(Map).length).toBe(1);
   });
```

Comment:
Please write the test cases of other functions that are now in use after switching to hooks.